PlayerInfo/PlayerInfo.cpp

```diff
@@ -62,6 +77,23 @@ namespace Plugin {
         if (_player == nullptr) {
             message = _T("PlayerInfo could not be instantiated.");
         }
+
+        try
```

Comment:
Exception handling is an expensive feature. This is the embedded environment. We have exception handling by default turned off. Sofar we had never a need for it. You could also return a status from the Utils::IARM::init inidicating success or failre.